woocommerce/wc-admin
# Analytics: Add toggle to disable UI features

Add a Feature toggle for Analytics. This PR is only responsible for disabling UI portions of the Feature. Note that disabling the REST API endpoints, data stores, and order processing will occur in follow up pull requests. 

### Screenshots

<img width="835" alt="Screen Shot 2021-06-15 at 12 44 12 PM" src="https://user-images.githubusercontent.com/1922453/121976184-6882d700-cdd7-11eb-9342-05bb6bd41545.png">

### Detailed test instructions:

1. Visit WooCommerce > Settings > Advanced > Features 
2. Play around with the toggles. Removing Analytics should remove Navigation items with and without the new Navigation turned on.
3. Confirm that visiting a link directly, ie `/wp-admin/admin.php?page=wc-admin&path=%2Fanalytics%2Fvariations` does not work.
